src/components/DetailedServerTile/DetailedServerTile.tsx

```diff
@@ -81,15 +81,15 @@ export class DetailedServerTile extends React.PureComponent<IDetailedServerProps
                 {this.props.status === ServerStatus.Offline &&
                     <div className={'counters-container'}>
                         <div className="offline-server-message">
-                            <Icon iconName={'icon-details'}></Icon>
+                            <Icon iconName={'icon-Info_krug'}></Icon>
```

Comment:
@rivisic @marijanaSpajic  well, this just feels wrong. naming icons in croatian. Or is this just a revert to the old names?